Contributing.md

```diff
@@ -107,7 +107,22 @@ move forward towards a consensus. It is not expected that a meeting of the TC
 will resolve all issues on its agenda during that meeting and may prefer to continue
 the discussion happening among the committers.
 
-Members can be added to the TC at any time. Any committer can nominate another committer
+Members can be added to the TC at any time. Any TC member can nominate another committer
 to the TC and the TC uses its standard consensus seeking process to evaluate whether or
-not to add this new member. Members who do not participate consistently at the level of
-a majority of the other members are expected to resign.
+not to add this new member. The TC will consist of a minimum of 3 active members and a
+maximum of 10. If the TC should drop below 5 members the active TC members should nominate
```

Comment:
Since the nomination starts at less than 5, should we also set minimum to 5?

Reply:
We discussed that a little bit on the call and thought we could just up it to 5 when we are regularly having more than 5 active members but just in case didn't want to start there and not be able to meet it. Happy if folks think we can just immediately start with 5 though.

Reply:
I can see why there is both a minimum of three but having the call to action to nominate at below five. I think the current wording is fine, as it really should have a reasonable minimum (and three seems reasonable to start with) but the call to nominate ideally should be out ahead of that like it is here.

Reply:
Makes sense, so we have an action item at 5 and know what happens then. But what about at 3? That part isn't clear.

Reply:
I am not sure what we can do in the governance wording to help with that. The wording here is with the hopes of never getting down that low. If we do then I guess the hope is new folks show up, but I don't think we can do much in this doc that would help prevent that can we? Happy to hear ideas, but I also think if we come up with good ideas we can amend this but not block merging for now.

Reply:
Yeah, I believe we can merge this for now and ammend later as necessary.

Reply:
Ok I'll call this resolved for now, but I would love to hear if anyone has ideas on how to improve the language to add some level of "how to resuscitate" lol. Technically I guess that is what we are doing now, so maybe one way we could do it is give the foundation CpC power to appoint someone if we are less than three. But to do something like that we need buy in from the CoC so I really think that is a separate issue/PR anyway.

Reply:
Ya. It is a good question, one we can punt for later. Because it seems either someone has to step in and make them be at least three if it were to drop to two or one or the TC would automatically dissolve if it happened, idk. That's why it probably makes sense to punt that got another pass as I think there are open questions around it to just slap in to here :)